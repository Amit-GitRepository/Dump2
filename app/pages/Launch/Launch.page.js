import Launch from '../../components/Launch/Launch.component';
import React, {Component} from 'react';

class LaunchPage extends Component {
  render () {
    return (
      <Launch />
    );
  }
}

export default LaunchPage;
