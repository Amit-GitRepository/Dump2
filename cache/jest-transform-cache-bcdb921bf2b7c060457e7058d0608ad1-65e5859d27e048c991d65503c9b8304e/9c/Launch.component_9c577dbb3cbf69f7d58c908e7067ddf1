Object.defineProperty(exports,"__esModule",{value:true});var cov_13o0xjpv4l=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/Launch/Launch.component.js',hash='62c535694f56884667c028d85e192124cb309829',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/Launch/Launch.component.js',statementMap:{'0':{start:{line:6,column:4},end:{line:8,column:6}}},fnMap:{'0':{name:'(anonymous_0)',decl:{start:{line:5,column:2},end:{line:5,column:3}},loc:{start:{line:5,column:12},end:{line:9,column:3}},line:5}},branchMap:{},s:{'0':0},f:{'0':0},b:{},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();var _react=require('react');var _react2=_interopRequireDefault(_react);
var _reactNative=require('react-native');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var

Launch=function(_Component){_inherits(Launch,_Component);function Launch(){_classCallCheck(this,Launch);return _possibleConstructorReturn(this,(Launch.__proto__||Object.getPrototypeOf(Launch)).apply(this,arguments));}_createClass(Launch,[{key:'render',value:function render()
{cov_13o0xjpv4l.f[0]++;cov_13o0xjpv4l.s[0]++;
return(
_react2.default.createElement(_reactNative.View,null));

}}]);return Launch;}(_react.Component);exports.default=


Launch;