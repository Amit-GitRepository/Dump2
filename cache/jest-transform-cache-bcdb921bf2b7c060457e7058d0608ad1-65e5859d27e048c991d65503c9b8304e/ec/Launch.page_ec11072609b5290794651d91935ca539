Object.defineProperty(exports,"__esModule",{value:true});var cov_2qfdtsdlhx=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/pages/Launch/Launch.page.js',hash='0570c7e653df0f6fcdef40e16983c48479d5e3d8',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/pages/Launch/Launch.page.js',statementMap:{'0':{start:{line:6,column:4},end:{line:8,column:6}}},fnMap:{'0':{name:'(anonymous_0)',decl:{start:{line:5,column:2},end:{line:5,column:3}},loc:{start:{line:5,column:12},end:{line:9,column:3}},line:5}},branchMap:{},s:{'0':0},f:{'0':0},b:{},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();var _Launch=require('../../components/Launch/Launch.component');var _Launch2=_interopRequireDefault(_Launch);
var _react=require('react');var _react2=_interopRequireDefault(_react);function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var

LaunchPage=function(_Component){_inherits(LaunchPage,_Component);function LaunchPage(){_classCallCheck(this,LaunchPage);return _possibleConstructorReturn(this,(LaunchPage.__proto__||Object.getPrototypeOf(LaunchPage)).apply(this,arguments));}_createClass(LaunchPage,[{key:'render',value:function render()
{cov_2qfdtsdlhx.f[0]++;cov_2qfdtsdlhx.s[0]++;
return(
_react2.default.createElement(_Launch2.default,null));

}}]);return LaunchPage;}(_react.Component);exports.default=


LaunchPage;