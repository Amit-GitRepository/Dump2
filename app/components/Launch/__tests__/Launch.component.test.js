import Launch from '../Launch.component';
import React from 'react';
import ReactSixteenAdapter from 'enzyme/build/adapters/ReactSixteenAdapter';
import renderer from 'react-test-renderer'; // Enzyme adapter for React 16
import {configure, shallow} from 'enzyme';

configure({adapter: new ReactSixteenAdapter()});

describe('Launch component', () => {
  const wrapper = shallow(<Launch/>);
  it('renders correctly', () => {
    const LaunchTree = renderer.create(<Launch />).toJSON();
    expect(LaunchTree).toMatchSnapshot();
  });
  it('renders correctly', () => {
    expect(wrapper).toBeDefined();
  });
});
