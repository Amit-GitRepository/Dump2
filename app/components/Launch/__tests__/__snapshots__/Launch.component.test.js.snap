// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Launch component renders correctly 1`] = `<View />`;
